MapboxCoreNavigation/LegacyRouteController.swift

```diff
@@ -300,11 +301,14 @@ open class LegacyRouteController: NSObject, Router, InternalRouter, CLLocationMa
             delegate?.router?(self, didUpdate: progress, with: location, rawLocation: rawLocation)
             
             //Fire the notification (for now)
-            NotificationCenter.default.post(name: .routeControllerProgressDidChange, object: self, userInfo: [
-                RouteControllerNotificationUserInfoKey.routeProgressKey: progress,
-                RouteControllerNotificationUserInfoKey.locationKey: location, //guaranteed value
-                RouteControllerNotificationUserInfoKey.rawLocationKey: rawLocation //raw
-                ])
+            var userInfo: [RouteControllerNotificationUserInfoKey: Any] = [
+                .routeProgressKey: progress,
+                .locationKey: location, //guaranteed value
+                .rawLocationKey: rawLocation, //raw
+            ]
+            userInfo[.headingKey] = heading
+            userInfo[.rawHeadingKey] = heading // heading snapping not yet implemented (an unnecessary?)
```

Comment:
YAGNI?

Reply:
We could get rid of `rawHeadingKey`, but that could be problematic if we ever do need to implement heading snapping for some reason. We’d want the non-`raw` values to be snapped when a `raw` value is available. That said, the comment refers to the prospect that heading snapping may not be especially useful.